ADAL/src/ADAuthenticationResult+Internal.m

```diff
@@ -216,6 +216,18 @@ + (ADAuthenticationResult *)resultFromBrokerResponse:(NSDictionary *)response
     ADTokenCacheItem* item = [ADTokenCacheItem new];
     [item setAccessTokenType:@"Bearer"];
     BOOL isMRRT = [item fillItemWithResponse:response];
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS1131)